generate_dragenScripts.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+if module list | grep -o -P 'NGS_DNA(.+)' 
```

Comment:
```suggestion
if module list -t NGS_DNA | grep NGS_DNA 
```

Reply:
your suggestion is not giving the same results:
```
module list -t NGS_Demultiplexing | grep NGS_Demultiplexing
1) NGS_Demultiplexing/2.3.1
```
vs

```
module list | grep -o -P 'NGS_Demultiplexing(.+)'
NGS_Demultiplexing/2.3.1
```

Reply:
Sorry, `-t` was in the wrong order; option must be added before `list` sub-command.I have updated the suggestion.

Reply:
You dropped the `grep` here, but `grep` is required to exit with non zero exit code if NGS_DNA was not loaded. `module list` will not fail no matter whether NGS_DNA is loaded or not.

Reply:
correct, I was too quick